source/server-discovery-and-monitoring/tests/unified/backpressure-server-description-unchanged-on-min-pool-size-population-error.yml

```diff
@@ -12,6 +12,20 @@ createEntities:
 tests:
   - description: the server description is not changed on handshake error during minPoolSize population
     operations:
+      - name: failPoint
```

Comment:
We should configure the failPoint before creating the testing client, because otherwise connection pool could create some connections before the failPoint will be configured.